python/ray/workers/default_worker.py

```diff
@@ -57,6 +57,12 @@
     type=str,
     default=None,
     help="Specify the path of the temporary directory use by Ray process.")
+parser.add_argument(
+    "--load-code-from-local",
+    required=False,
+    type=bool,
```

Comment:
You can use `action='store_true'` to make it so that you just pass in `--load-code-from-local` instead of `--load-code-from-local=True`